spec/status.xml

```diff
@@ -8,47 +8,25 @@ technical report can be found in the <loc
+This document is intended to become a W3C Recommendation. Comments regarding this document are welcome. 
+Please send them to <loc href="mailto:public-tt@w3.org">public-tt@w3.org</loc> 
+(<loc href="mailto:public-tt-request@w3.org">subscribe</loc>, 
+<loc href="https://lists.w3.org/Archives/Public/public-tt/">archives</loc>) with <code>[ttml13ed]</code> 
+at the start of your email's 
+subject. W3C publishes a Candidate Recommendation to indicate that the document is believed to be stable 
+and to encourage implementation by the developer community. This Candidate Recommendation is expected to 
+advance to Proposed Recommendation no earlier than XXXXX.</p>
 
-<p>This version reflects changes made as a result of applying errata and addressing
-comments received since the Recommendation was first published. With one minor
-exception noted below, these changes are editorial.</p>
+<p>This new edition of the Timed Text Markup Language (TTML) 1.0
+<bibref ref="ttml10"/> supersedes the <loc href="http://www.w3.org/TR/2013/REC-ttml1-20130924/">previous
+edition</loc>. A list of the substantive changes applied since that previous edition is found at <loc href="ttml1-changes.html">ttml1-changes.html</loc>. No feature was added.</p>
 
-<p>This document is a revision of Timed Text Markup Language (TTML) 1.0
-<bibref ref="ttml10"/>, superseding that <loc href="http://www.w3.org/TR/2010/REC-ttaf1-dfxp-20101118/">previous
-Recommendation</loc>. The only identified substantive difference between this version and
-its predecessor is a change of the default treatment of the <att>ttp:markerMode</att> parameter attribute
-from <code>continuous</code> to <code>discontinuous</code>. A cumulative summary of all changes applied
-to this version since the Recommendation was first published is available at
-<loc  href="ttml1-changes.html#change-history-rec-to-2e-per">Changes
-from Recommendation to Proposed Edited Recommendation (2nd Ed.)</loc> and
-<loc  href="ttml1-changes.html#change-history-2e-per-to-rec-2e">Changes
-from Proposed Edited Recommendation (2nd Ed.) to Recommendation (2nd Ed.)</loc>.</p>
+<p>For this specification to exit the CR stage, at least 2 independent implementations of every <loc>CR exit criteria test</loc> is required, as documented in the Working Group's <loc>implementation report</loc>. The Working Group does not require that implementations are publicly available but encourages them to be so.</p>
 
-<p>A <loc href="http://www.w3.org/2008/10/dfxp-testsuite.zip">sample test suite</loc> is available. This set of tests is not intended to
-be exhaustive. The <loc href="http://www.w3.org/2009/05/dfxp-results.html">implementation report</loc> used by the director to
-transition to Recommendation has been made available. There have been no formal objections
-to the publication of this document.</p>
+<p>The Working Group has not identified features "at risk" for this specification.</p>
 
-<p>This document was produced by the <loc
```

Comment:
Any reason why this paragraph is removed?

Reply:
It was moved above

Reply:
Do you mean moved to the new line 11 ("This document was published by the Timed Text Working Group ...")? That looks very different to me.

Reply:
I used the boilerplate found in IMSC1.0.1 (and respec.js) figuring it was more up to date.

Reply:
Do the pubrules say anything about this? I'm concerned that we previously acknowledged explicitly the contribution of SMPTE and EBU and that by removing those acknowledgements it could be taken as a signal even if we don't intend it as such. So if we have freedom to write this as we wish, I would reinstate it.

Reply:
@nigelmegitt The SOTD is by nature transitory. If something is core to the nature document, it should be listed in the scope. Contributions can be listed in the acknowledgement section.